src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs

```diff
@@ -51,13 +51,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<Frame> e)
 
 			if (e.NewElement != null)
 			{
-				_actualView.CrossPlatformArrange = (e.NewElement as IContentView).CrossPlatformArrange;
-				_actualView.CrossPlatformMeasure = (e.NewElement as IContentView).CrossPlatformMeasure;
+				_actualView.CrossPlatformLayout = e.NewElement;
 
 				SetupLayer();
 				UpdateShadow();
 			}
-		}
+		} 
 
```

Comment:
looks like a spurious space got added after the '}'

Reply:
That's what the formatting bot is for :)